internal/oci/runtime_oci.go

```diff
@@ -712,6 +712,18 @@ func (r *runtimeOCI) StopContainer(ctx context.Context, c *Container, timeout in
 		}
 	}()
 
+	// Paused containers cannot be stopped, so we have to resume them and
+	// update the container status accordingly.
+	if c.state.Status == ContainerStatePaused {
+		logrus.Warnf("Container %s is paused, resuming before trying to stop", c.ID())
+		if err := r.UnpauseContainer(ctx, c); err != nil {
+			return errors.Wrap(err, "resume container")
```

Comment:
would it be better to have "failed to resume container"?  If so, similar comment for other error messages.

Reply:
Well' I'd prefer reducing the amount of word repetitions in error messages. Usually the return value (an `error` type) already indicates that something went wrong. Assuming that those errors are chained, the resulting message would look like:
`error in x: did y: on my way to z` compared to: `error in x: failed to do y: failed to z`